### PR TITLE
--Updated the code for code review comments.

### PR DESCRIPTION
**Pull request by vennkataraomindtree**

### CODE REVIEW

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/dto/3dsStepOne.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+var AbstractResponse = require('~/cartridge/scripts/dto/base/abstractResponse');
+var ThreeDs = require('~/cartridge/scripts/dto/nested/threeDs');
+var Order = require('~/cartridge/scripts/dto/nested/order');
+var PaymentMethod = require('~/cartridge/scripts/dto/nested/paymentMethod');
+var BrowserData = require('~/cartridge/scripts/dto/nested/browserData');
+/**
+ * Forms all the fields required to send for Authentication request.
+ * @param {obj} requestObj - object that contains fields for request to be sent.
+ */
+ var ThreeDsSteponeRequest = AbstractRequest.extend({
```
Comment (mandal-tanmay) [inline Link_globalPay/cartridges/int_globalpay/cartridge/scripts/dto/3dsStepOne.js:13]: ThreeDsSteponeRequest  to ThreeDsStepOneRequest may be

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
 'use strict';
```
Comment (mandal-tanmay) [inline Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js:1]: creditcardutils.js to creditCardUtils may be?

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
 
+
+/**
+ * for authenticating the given pmt_token.
+ * @param req -- request object
+ * @param res -- resp object
+ * @return Object -- authentication object
+ */
 function authenticationData(req, res) {
+  var myreq = req.httpParameterMap.requestBodyAsString;
```
Comment (mandal-tanmay) [inline Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js:18]: myReq is a not a good variable name - Lets use requestBodyAsString for this/

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/services/globalPayService.js [hunk 4]
```diff
@@ -68,7 +68,7 @@ function executeRequest(requestObject, responseClass) {
   };
 
   if (serviceResult.getErrorMessage()) {
-    var APIError = require('*/cartridge/scripts/dtos/APIError');
+    var APIError = require('*/cartridge/scripts/dto/apiErrors');
```
Comment (mandal-tanmay) [inline Link_globalPay/cartridges/int_globalpay/cartridge/scripts/services/globalPayService.js:71]: The callback filterLogMessage or both getRequestLogMessage and getResponseLogMessage should be implemented for every service definition.

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/services/globalPayService.js [hunk 5]
```diff
@@ -68,7 +68,7 @@ function executeRequest(requestObject, responseClass) {
...
```
Comment (mandal-tanmay) [inline Link_globalPay/cartridges/int_globalpay/cartridge/scripts/services/globalPayService.js:72]: No To-do

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/services/globalPayService.js (hunk 4 above)
Comment (vennkataraomindtree) [reply Link_globalPay/cartridges/int_globalpay/cartridge/scripts/services/globalPayService.js:70]: alery we have impleted filterLogMessage(), please suggest us if we need to implemenet any another way.

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js (hunk 2 above)
Comment (vennkataraomindtree) [reply Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js:1]: changed as suggested.

File: Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js (hunk 3 above)
Comment (vennkataraomindtree) [reply Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js]: This has changed as you suggested.

Comment (vennkataraomindtree) [reply Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js]: This has changed as you suggested.

Comment (vennkataraomindtree) [reply Link_globalPay/cartridges/int_globalpay/cartridge/scripts/util/creditcardutils.js]: changes as suggested.

**Pull request merged**